### PR TITLE
texlive-bin: update to 2019.

### PR DESCRIPTION
**Pull request by SuperNascher**

### CODE REVIEW

**Comment by abenson** [comment]
Don't forget to edit the PATH in `/etc/profile.d/texlive.sh`

**Comment by leahneukirchen** [comment]
Let's pause this for a week until really all CTAN mirrors caught up (which was not the case as of yesterday.)

**Comment by SuperNascher** [comment]
> Don't forget to edit the PATH in `srcpkgs/texlive-bin/files/texlive.sh`

That is not needed, because xbps changes the script to the latest version.
https://github.com/void-linux/void-packages/blob/master/srcpkgs/texlive-bin/files/texlive.sh

File: srcpkgs/texlive-bin/template [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Template file for 'texlive-bin'
 pkgname=texlive-bin
-version=2018
+version=2019
 revision=1
+depends="texlive${version}-bin"
+short_desc="TeX Live Binary distribution through tl-install (newest version)"
 maintainer="Leah Neukirchen <leah@vuxu.org>"
-homepage="http://tug.org/texlive/"
 license="GPL-2"
```
Comment (abenson) [inline srcpkgs/texlive-bin/template]: This should be updated as well to use SPDX

**Pull request merged**